### ISSUE TITLE
Error while syncing with old (free) G Suite accounts

### ISSUE THREAD

**Issue by finalwebsites** [Foundry376/Mailspring#1630]
Hi,

I get every time a sync error if I start the mail client for two of my accounts:

The log files are showing this info:

```
27730 [2019-09-07 11:22:00.002] [metadata] [info] Metadata delta stream starting...
27730 [2019-09-07 11:22:00.002] [background] [info] Fetching XOAuth2 access token from Gmail for 29e49caf
27730 [2019-09-07 11:22:00.003] [metadata] [info] Metadata delta stream closed.
27730 [2019-09-07 11:22:00.012] [background] [critical] 
 ***
 *** A JSON exception occurred during program execution: 
 *** [json.exception.type_error.316] invalid UTF-8 byte at index 0: 0xA8
 ***

27730 [2019-09-07 11:22:00.016] [background] [critical]  *** Stack trace:
 ***   clone
 ***
```
I'm using Mailspring on Ubuntu 18.04 and both mail accounts are old free G Suite accounts. 
It's not a problem to sync the data after I re-authorized the connection.

There is also a Gmail account which doesn't have these problems.

**Comment by rebeccaskinner** [comment]
I'm experiencing the same issue with a current paid G Suite account on Pop!_OS 19.04 and mailspring 1.6.3-134a0e55 installed via snap.

**Comment by bengotow** [comment]
Hey folks - just a quick heads up that the `invalid UTF-8 byte at index 0` bug will be fixed in the next release this week. This turned out to be a tricky one and the app doesn't retry the connection because the error is actually thrown within the error handling logic for the sync worker. Stay tuned!

**Comment by bengotow** [comment]
Hey folks! This shipped today in 1.7.2 - you can download the update from our website at getmailspring.com/download or choose "Check for Updates" in the app's menu. Enjoy the update!
